Up2dateService/Up2dateService/Installers/Choco/ChocoInstaller.cs

```diff
@@ -55,31 +68,43 @@ public bool IsPackageInstalled(Package package)
 
         public void Refresh()
         {
-            using (var ps = PowerShell.Create())
-            {
-                const string psCommand = @"choco list -li";
-                ps.AddScript(psCommand);
-                productCodes.Clear();
-                productCodes.AddRange(ps.Invoke<string>());
-            }
+            Process p = new Process();
+            p.StartInfo.FileName = "choco.exe";
+            p.StartInfo.Arguments = "list -l";
+            p.StartInfo.RedirectStandardOutput = true;
+            p.StartInfo.UseShellExecute = false;
+            p.Start();
+            p.WaitForExit();
+            productCodes.Clear();
+            productCodes.AddRange(p.StandardOutput.ReadToEnd()
+                .Split(new[] { Environment.NewLine },
```

Comment:
Are you sure such split does the right job?
Sometimes (depends on application) the output may contain CR+LF sequence at the end of each line.
In this case the elements would still contain CR at the end which is not what we expect.

I would implement it in more clear and straight-forward way using StandardOutput.ReadLine() method flitering out the lines which string.IsNullOrWhiteSpace()